src/webhook_launcher/app/models.py

```diff
@@ -340,13 +340,18 @@ def clean(self, exclude=None):
         self.project = self.project.strip()
         self.package = self.package.strip()
 
-        if WebHookMapping.objects.exclude(pk=self.pk).filter(
+        duplicates = WebHookMapping.objects.filter(
             project=self.project,
             package=self.package,
-            obs=self.obs
-        ).count():
+            obs=self.obs,
```

Comment:
shouldn't this include build=True  .... and/or placeholder=false

Otherwise I may have turned off build on a previously building webhook, created a new one and can't set build on it?